src/classes/Keyboard.cpp

```diff
@@ -830,19 +847,25 @@ bool LedKeyboard::sendDataInternal(byte_buffer_t &data) {
 			return true;
 		#elif defined(libusb)
 			if (! m_isOpen) return false;
-			if (data.size() > 20) {
-				if(libusb_control_transfer(m_hidHandle, 0x21, 0x09, 0x0212, 1, 
-						const_cast<unsigned char*>(data.data()), data.size(), 2000) < 0)
-					return false;
-			} else {
-				if(libusb_control_transfer(m_hidHandle, 0x21, 0x09, 0x0211, 1, 
-						const_cast<unsigned char*>(data.data()), data.size(), 2000) < 0)
-					return false;
-			}
+
+			uint16_t wValue;
+			unsigned char endpoint = 0x82;
+
+			if (data.size() > 20)
+				wValue = 0x0212;
+			else if (data.size() < 20) {
+				wValue = 0x0210;
+				endpoint = 0x83;
```

Comment:
We are not currently use this in the current code, but it could be useful in the future as we implement more features. At least I've seen such messages with tcpdump.